boards/arm/stm32f072b_disco/stm32f072b_disco.dts

```diff
@@ -48,11 +48,20 @@
 		};
 	};
 
+	pwmleds: pwmleds {
+		compatible = "pwm-leds";
+
```

Comment:
better add 
`status = "disabled";`
since the overlay enables the node

Reply:
Done.